src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -193,6 +191,8 @@ private class Emit(
         present(const(true))
       case False() =>
         present(const(false))
+      case Void() =>
+        EmitTriplet(Code._empty, const(false), Code._empty)
```

Comment:
just something we now have to keep in mind: all IR nodes either:
 - add one thing to the JVM stack
 - add nothing to the JVM stack

Reply:
or I guess `Die` just ends the program.

And aggregators have always been a bit weird in this context...

Reply:
`Code[Unit]` always pushed nothing on the stack as far as I know.